backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/nuernberg/steps/DownloadCsv.kt

```diff
@@ -25,14 +25,14 @@ class DownloadCsv(config: ImportConfig, private val logger: Logger) :
             )
             val parser = CSVFormat.RFC4180.builder().setHeader().setSkipHeaderRecord(true).build().parse(inputCSV)
 
-            return getCSVAcceptingStores(parser)
+            return getCSVAcceptingStore(parser)
         } catch (e: Exception) {
             logger.error("Unknown exception while downloading data from csv", e)
             throw e
         }
     }
 
-    private fun getCSVAcceptingStores(parser: CSVParser): List<CSVAcceptingStore> {
+    private fun getCSVAcceptingStore(parser: CSVParser): List<CSVAcceptingStore> {
```

Comment:
why renamed? if it returns a list of stores

Reply:
yes, my bad